crates/orchestrator/src/tests/config.rs

```diff
@@ -64,116 +93,347 @@ impl TestConfigBuilder {
             queue: None,
             storage: None,
             alerts: None,
+
+            database_node: None,
+            data_storage_node: None,
+            data_storage_client: None,
+            queue_node: None,
+            queue_client: None,
+            alert_client: None,
+            alert_node: None,
         }
     }
 
-    pub fn mock_da_client(mut self, da_client: Box<dyn DaClient>) -> TestConfigBuilder {
+    pub async fn testcontainer_s3_data_storage(mut self) -> TestConfigBuilder {
```

Comment:
the implementations looks clean. maybe we can replace `testcontainer_s3_data_storage` with `connect_to_s3_storage` and similar for others

Reply:
I would like to go with `connect_to_new_s3_storage` to let know that the instance created will be new and clean.
WDYT ?